include/mbedtls/config.h

```diff
@@ -2133,7 +2133,7 @@
  * Enable the platform abstraction layer that allows you to re-assign
  * functions like calloc(), free(), snprintf(), printf(), fprintf(), exit().
  *
- * Enabling MBEDTLS_PLATFORM_C enables to use of MBEDTLS_PLATFORM_XXX_ALT
+ * Enabling MBEDTLS_PLATFORM_C enables use of MBEDTLS_PLATFORM_XXX_ALT
```

Comment:
The first sentence sounds slightly odd: `Enabling MBEDTLS_PLATFORM_C enables`. Perhaps it should be changed to something like `Defining MBEDTLS_PLATFORM_C enables...`